configuration/multimedia

```diff
@@ -1,136 +1,160 @@
+Multimedia
+==========
 
-Listen to Spotify
-=================
-Install-Howto: https://www.spotify.com/de/download/linux/
+In order to choose Qubes OS as your primary OS it must be able to do all tasks, including playing multimedia content.
+This howto explains how to create a multimedia temmplate which can be used to play multimedia content.
+This includes:
 
-Detailed steps which must be run in the Debian 8 template VM.
-A good idea might be to clone the existing Debian 8 template and create an own debian-8-multimedia template from whoch you can then create an App-VM.
-You need to make sure that you allow networking to the internet during installation, as you need to download some data which are not from the default repositories.
+- Spotify
+- Amazon Prime
+- Netflix
+- DVDs
 
-# Become root
-su -
-# Download the Key which is used for signing the Spotify package repository
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys BBEBDCB318AD50EC6865090613B00F1FD2C19886
-#Public-Key: http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0xD2C19886&fingerprint=on
-#Fingerprint: BBEB DCB3 18AD 50EC 6865 0906 13B0 0F1F D2C1 9886
-# Add the package repository to your list of package sources
-echo deb http://repository.spotify.com stable non-free | tee /etc/apt/sources.list.d/spotify.list
-# Make the new repository known to your system
-apt-get update
-# Install the Spotify Client
-apt-get install -y spotify-client
+Hint: This first draft of this howto was written under Qubes OS 3.2 but it should also work for Qubes 4rc4.
 
 
+Installation
+------------
 
-Play Netflix
-============
-You can play Netflix by installing a more recent version of firefox or by installing Google Chroome.
-As Google Chrome will play video from Netflx and (!) Amazon, the suggestion is to install Google Chrome.
-If you don't use Amazon Prime, Firefox will work.
+Start by cloning the default debian template in dom0.
+Hint:
+t-multimedia is just the template VM where we will install all packages.
+In the last step we will create an AppVM from this template.
 
-Install Firefox
----------------
-The built in Firefox (from the Debian Template VM) was Firefox ESR 45.3.0
-According to the "supported browser" list Mozilla Firefox >= 47.x is needed.
-I've thereoff installed Firefox v53.0.2 according to this manual:
-http://libre-software.net/how-to-install-firefox-on-ubuntu-linux-mint/
+`qvm-clone debian-9 t-multimedia`
 
-1) download firefox from https://www.mozilla.org/en-US/firefox/new/?scene=2
-direct download link:
-https://download-installer.cdn.mozilla.net/pub/firefox/releases/53.0.2/linux-x86_64/en-US/firefox-53.0.2.tar.bz2
+As we need to install some package outside of the regular repositories you need to enable networking for the Template VM.
 
-2) unpack the downloaded file
-tar -xjf firefox-53.0.2.tar.bz2
+`qvm-prefs --set t-multimedia netvm sys-firewall`
```

Comment:
These steps shouldn't normally be necessary, since the TemplateVM should already have network access (at least to the updates proxy).

Reply:
This information is old, as the new part of the howto is based on Chrome.
I have attached the multimedia.md file, so that you can review it.
[multimedia.md.txt](https://github.com/QubesOS/qubes-doc/files/1735244/multimedia.md.txt)

Reply:
Please include any new changes in the PR. We don't review separate attached files.